thriftserver/README.md

```diff
@@ -0,0 +1,73 @@
+# Spark Thrift Server - HiveServer2
+
+Spark Thrift Server component installs HiveServer2 variant for Spark SQL - Thriftserver. It deploys the Spark SQL Thrift Server intended to expose Spark dataframes modeled as Hive tables through a JDBC connection.
+
+### Folders
+
+There is one main folder in the Thrift Server component `thriftserver` which contains the kustomize manifests.
+
+### Installation
+
+To install Thrift Server add the following to the `kfctl` yaml file.
+
+Minimal install:
+
+```yaml
+- kustomizeConfig:
+    parameters:
+      - name: spark_url
```

Comment:
Is `spark_url` commonly used name? I've never seen Spark connection specified by URL so far (although I only saw them in OpenShift environment where that is not easily possible due to route limitations)

Reply:
The point being, should this be something like `spark_endpoint` to not suggest it should be a DNS name?

Reply:
@vpavlin It's usually referred to as as *master url*. Endpoint itself is not sufficient, because it has to also contain the used protocol like `spark://` or `yarn://` etc. 

See this reference:
https://spark.apache.org/docs/latest/submitting-applications.html#master-urls

Maybe `spark_master_url` would work better?

Reply:
The goal is mainly to make sure people understand what to put in that value

Reply:
I've left the variable name as it and updated the documentation. WDYT?

Reply:
Sounds good